app/views/moderatorIndex.pug

```diff
@@ -0,0 +1,28 @@
+extends layout
+
+block scripts
+    script(src='/static/bundle.js')
+
+block navbar
+    nav
+        ul.list-unstyled
+            li
+                a(href='/moderator/create_user') Registrer bruker
+            li
+                a(href='/moderator/activate_user') Aktiver bruker
+            li
+                a(href='/moderator/change_card') Mistet kort
+            li
+                a(href='/moderator/deactivate_users') Deaktiver brukere
+
+block content
+    .center(ng-repeat='alert in alerts', ng-class='{"fade": alert.fade}')
+        uib-alert(ng-bind='alert.message', type='{{alert.type}}', close='alert.close()')
+    .container(ng-view='')
+
+block footer
```

Comment:
Maybe move this to a generic view?

Reply:
Smart, this was quite easy as well. A lot of files had the same block footer but it could just be moved over to the `layout.pug`, a file which all extends.